gameview.lua

```diff
@@ -160,10 +160,13 @@ end
 
 function GameView:leftChar()
    if (isTap == 1 and self.active_row_num == 1 and self.active_col_num == 1) then
+      isTap = 0
       return
    elseif (isTap == 1 and self.puzzle.size.cols > 15 and self.active_row_num < 3 and self.active_col_num == 1) then
```

Comment:
What's the reason for this `elseif` conditional? I was not aware there are issues on larger puzzles! :o